recipe/meta.yaml

```diff
@@ -125,6 +131,8 @@ outputs:
         - pyqt >=5.15,<5.16
         - pyqtwebengine >=5.15,<5.16
         - qtconsole >=5.6.1,<5.7.0
+        # https://github.com/conda-forge/fcitx-qt5-feedstock/issues/7
+        - fcitx-qt5 >=1.2.7  # [linux]
```

Comment:
This package was causing `qt5-main` to be required on all systems even for `spyder-base` so I moved it to the `spyder` package rather than `spyder-base` and opened an upstream issue

Reply:
Great catch! @mrclary, could you take care of this in another PR?